src/base/projects/View.svelte

```diff
@@ -16,11 +16,13 @@
   export let urn: string;
   export let org = "";
   export let user = "";
-  export let commit = "";
   export let config: Config;
-  export let path: string;
+  export let peer: string;
+  export let location: any;
 
-  let content = proj.ProjectContent.Browser;
+  let content = location.pathname.indexOf("history") !== -1 ? proj.ProjectContent.History : proj.ProjectContent.Tree;
```

Comment:
I think there is a better way to check for the current ProjectView. Will improve

Reply:
Okay I found a more robust way by creating a regex that matches the available paths.